Userland/Libraries/LibSQL/Database.cpp

```diff
@@ -47,6 +48,21 @@ ErrorOr<void> Database::open()
         default_schema = SchemaDef::construct("default");
         TRY(add_schema(*default_schema));
     }
+
+    auto master_schema = TRY(get_schema("master"));
+    if (!master_schema) {
+        master_schema = SchemaDef::construct("master");
+        TRY(add_schema(*master_schema));
+    }
+
+    auto describe_internal_table = TableDef::construct(master_schema, "internal_describe_table");
+    describe_internal_table->append_column("Name", SQLType::Text);
+    describe_internal_table->append_column("Type", SQLType::Text);
+
+    // `add_table` errors only on recreating an already-existing table,
+    //  which we don't care about.
+    auto discarded_return_value = add_table(*describe_internal_table);
```

Comment:
I'm kinda surprised this doesn't also raise a warning about an unused variable. But I'm not sure we want to suppress errors here anyways?
```c++
TRY(add_table(*describe_internal_table));

Reply:
For some reason, the table gets created repeatedly (I don't exactly get why)... So using `TRY` caused problems as it errs and logs "duplicate table" or something similar.

Should I check if the table exists, and if not so create it? I manually checked `is_error` on the return value of `add_table` but it's not so meaningful as we want to ignore the error (`add_table` probably only errs on a recreation of an already existing table). So what do you suggest?

Reply:
I'd assume it already exists if the database file on disk already exists.

So on a brand new database, we successfully create the internal DESCRIBE table, then at some point later we persist that database to disk. Then when we "create" the database again from that file, the table already exists, and we get the error.

I'd suggest checking if it exists before adding it, that way we can propagate any errors creating the table up to the caller. There might not be any other errors now, but that could change in the future, and I'd rather not mask real errors.

just my opinion, @JanDeVisser can chime in if he has another opinion :)